src/Files.App/Views/ShellPanesPage.xaml.cs

```diff
@@ -176,23 +176,23 @@ public bool IsRightPaneVisible
 				if (value != _IsRightPaneVisible)
 				{
 					_IsRightPaneVisible = value;
```

Comment:
I suppose we still want to remove this property in a future PR.

Reply:
Definitely yes. But we have to do it slowly not in the same PR. I realized that in my previous trial 😅